diff --git a/src/private/webapis/dom/nodes/specs/NativeDOM.js b/src/private/webapis/dom/nodes/specs/NativeDOM.js
index 60e23b2efe98146d1e3ac5915d04c4d261836fed..bd7a686d81a5afcfa1749b113a351fae385ff325 100644
--- a/src/private/webapis/dom/nodes/specs/NativeDOM.js
+++ b/src/private/webapis/dom/nodes/specs/NativeDOM.js
@@ -17,8 +17,8 @@ import {getFabricUIManager} from '../../../../../../Libraries/ReactNative/Fabric
 import * as TurboModuleRegistry from '../../../../../../Libraries/TurboModule/TurboModuleRegistry';
 import nullthrows from 'nullthrows';
 
-export opaque type NativeElementReference = ShadowNode;
-export opaque type NativeTextReference = ShadowNode;
+export type NativeElementReference = ShadowNode;
+export type NativeTextReference = ShadowNode;
 
 export type NativeNodeReference =
   | NativeElementReference
diff --git a/src/private/webapis/idlecallbacks/specs/NativeIdleCallbacks.js b/src/private/webapis/idlecallbacks/specs/NativeIdleCallbacks.js
index 4241e5ef06d27bf41d919776eab94f1f12800681..0e5189127c2857ae687e4088453fc8a42a25bf5d 100644
--- a/src/private/webapis/idlecallbacks/specs/NativeIdleCallbacks.js
+++ b/src/private/webapis/idlecallbacks/specs/NativeIdleCallbacks.js
@@ -12,7 +12,7 @@ import type {TurboModule} from '../../../../../Libraries/TurboModule/RCTExport';
 
 import * as TurboModuleRegistry from '../../../../../Libraries/TurboModule/TurboModuleRegistry';
 
-export opaque type IdleCallbackID = mixed;
+export type IdleCallbackID = mixed;
 
 export type RequestIdleCallbackOptions = {
   timeout?: number,
diff --git a/src/private/webapis/intersectionobserver/specs/NativeIntersectionObserver.js b/src/private/webapis/intersectionobserver/specs/NativeIntersectionObserver.js
index af1cac035ab061b3e960d772dfae34b61a30985b..f9e83084bf8c2f4714ae350e731db4750b6568dc 100644
--- a/src/private/webapis/intersectionobserver/specs/NativeIntersectionObserver.js
+++ b/src/private/webapis/intersectionobserver/specs/NativeIntersectionObserver.js
@@ -31,7 +31,7 @@ export type NativeIntersectionObserverObserveOptions = {
   rootThresholds?: ?$ReadOnlyArray<number>,
 };
 
-export opaque type NativeIntersectionObserverToken = mixed;
+export type NativeIntersectionObserverToken = mixed;
 
 export interface Spec extends TurboModule {
   // TODO(T223605846): Remove legacy observe method
diff --git a/src/private/webapis/performance/specs/NativePerformance.js b/src/private/webapis/performance/specs/NativePerformance.js
index 18714050b15d820c1241a954c104af26bd783afe..809037e8a731e450b8581fa040b8dcb1842e6ebe 100644
--- a/src/private/webapis/performance/specs/NativePerformance.js
+++ b/src/private/webapis/performance/specs/NativePerformance.js
@@ -39,7 +39,7 @@ export type RawPerformanceEntry = {
   responseStatus?: number,
 };
 
-export opaque type OpaqueNativeObserverHandle = mixed;
+export type OpaqueNativeObserverHandle = mixed;
 
 export type NativeBatchedObserverCallback = () => void;
 export type NativePerformanceMarkResult = number;
